crypto/fipsmodule/ec/p384.c

```diff
@@ -561,7 +635,7 @@ static crypto_word_t fiat_p384_get_bit(const uint8_t *in, int i) {
 // It forces an odd scalar and outputs digits in
 // {\pm 1, \pm 3, \pm 5, \pm 7, \pm 9, ...}
 // i.e. signed odd digits with _no zeroes_ -- that makes it "regular".
-static void fiat_p384_mul_scalar_rwnaf(int16_t *out, const unsigned char *in) {
+static void p384_felem_mul_scalar_rwnaf(int16_t *out, const unsigned char *in) {
```

Comment:
I was wondering if you wanted to remove "fiat" from `fiat_p384_to_generic` and `fiat_p384_inv_square` as well.

Reply:
will do